### PR TITLE
FEATURE: Add coin divisibility to crypto listings

### PR DESCRIPTION
**Pull request by ghost**
#888

Depends on PR #887

### CODE REVIEW

File: core/listings.go [hunk 1]
```diff
@@ -43,6 +43,8 @@ const (
 	MaxCountryCodes          = 255
 	EscrowTimeout            = 1080
 	SlugBuffer               = 5
+
+	DefaultCoinDivisibility uint32 = 10e7
```
Comment (placer14) [inline core/listings.go]: Suppose to be 100_000_000 instead of 10_000_000.

Comment (placer14) [inline core/listings.go]: Suppose to be 100_000_000 instead of 10_000_000

**Review by placer14 — CHANGES_REQUESTED** [review]
One fix. GTM!

File: core/listings.go (hunk 1 above)
Comment (tyler-smith) [reply core/listings.go]: It is 100,000,00 but I can change it to 1e8 instead of 10e7 if that's more readable.

**Comment by tyler-smith** [comment]
@placer14 It wasn't incorrect, but I've changed it from 10e7 to 1e8 to be more clear.

**Review by placer14 — APPROVED** [review]

**Pull request merged**